### PR TITLE
ISPN-11684 when the creation is not ok, handle errors correctly

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-11684

backport #8

### CODE REVIEW

File: src/services/cacheService.ts [hunk 1]
```diff
@@ -105,17 +115,27 @@ class CacheService {
...
+          success: true
+        };
+      })
+      .catch(err =>
+        err
+          .text()
+          .then(
+            errorMessage =>
+              <ActionResponse>{ message: errorMessage, success: false }
+          )
```
Comment (ryanemerson) [inline src/services/cacheService.ts:138]: Small thing, but the response handling in `createCacheByConfigName` and `createCacheWithConfiguration` appears to be identical. Can they be combined into a single method that takes a `Promise<Response>` as a param?

File: src/services/cacheService.ts [hunk 2]
```diff
@@ -105,17 +115,27 @@ class CacheService {
...
+        if (response.ok) {
+          return response.text();
+        }
+        throw response;
+      })
+      .then(text => {
+        let sucessMsg =
+          text == '' ? 'Cache ' + cacheName + ' created with success' : text;
+        return <ActionResponse>{
+          message: sucessMsg,
```
Comment (ryanemerson) [inline src/services/cacheService.ts:128]: Perform the action inline?
` message: text == '' ? 'Cache ' + cacheName + ' created with success' : text,

**Comment by karesti** [comment]
@ryanemerson done. I don't know what happened yesterday

**Comment by ryanemerson** [comment]
Thanks @karesti

**Pull request merged**